src/components/content-display/markdown-content/markdown-content.scss

```diff
@@ -76,4 +98,43 @@
     bottom: 0;
     fill: var(--magentaa11y-color-fill-divider-brand);
   }
+
+  textarea, 
+  input[type="text"], 
+  input[type="search"], 
+  input[type="file"], 
+  input[type="email"], 
+  input[type="date"], 
+  input[type="tel"], 
+  input[type="number"], 
+  input[type="password"], 
+  input[type="button"] {
+    border: 2px solid black;
```

Comment:
can you get rid of the  input[type="search"] {} styles in md-styles.scss since this looks like a duplicate now?

Reply:
or at least some of the styles

Reply:
actually this maybe belongs in the md-styles file, scoped under example/.example